database/database_test.go

```diff
@@ -0,0 +1,133 @@
+package database
+
+import (
+	"fmt"
+	"testing"
+
+	obj "github.com/jcgallegdup/Concurrent-Document-Editor/objects"
+)
+
+const path = "../updates.db"
+const invalidID = 9001
+const ID = 1
+
+var expectedChanges = []obj.Change{obj.NewChange("insert", 0, "a"), obj.NewChange("insert", 1, "b")}
+
+func TestCreateUserReturnsIncrementedCounterReal(t *testing.T) {
```

Comment:
Uh yeah this one's a little awkward. Go doesn't allow duplicate functions with the same name to be exported, since a lot of these tests are replicated in the `mockDB_test.go` file there were naming conflicts. Do you have any suggestions for better names? Or maybe another solution?

Reply:
Oh yeah, that's partially my fault. If you want, you could change all my tests in `mockDB_test.go` to start with `TestMock` and then you could get rid of `Real` in your tests. I think that would make more sense.

Reply:
OH right, I see. Naming the mock ones differently might be slightly better, but I'll just leave it up to you